src/jsonUtils/nodeImpl/makeSvgLayer.js

```diff
@@ -1,9 +1,97 @@
+import type { SJRect, SJLayer, SJGroupLayer } from 'sketchapp-json-flow-types';
+import { BorderPosition } from 'sketch-constants';
 import type { LayoutInfo } from '../../types';
-import { makeRectShapeLayer } from '../shapeLayers';
+import { makeShapeGroup, makeShapePath } from '../shapeLayers';
+import { makeRect } from '../models';
+import { createUniformBorder } from '../borders';
+import layerGroup from '../layerGroup';
 
-// TODO:
-export default function makeSvgLayer(layout: LayoutInfo, name: string /* , svg: string */) {
-  const shape = makeRectShapeLayer(0, 0, layout.width, layout.height);
-  shape.name = name;
-  return shape;
+import { makePathsFromCommands, makeLineCapStyle } from './graphics/path';
+import { unionRects, scaleRect, makeBoundingRectFromCommands, resize } from './graphics/rect';
+
+const svgModel = require('@lona/svg-model');
```

Comment:
let's require it with the weird hack from here: https://github.com/airbnb/react-sketchapp/blob/master/src/jsonUtils/nodeImpl/requireNodobjC.js#L9

Reply:
cool, seems like it works